src/test/java/teammates/test/cases/action/BaseActionTest.java

```diff
@@ -731,4 +731,8 @@ protected void verifyNumberOfEmailsSent(Action action, int emailCount) {
         assertEquals(emailCount, action.getEmailSender().getEmailsSent().size());
     }
 
+    protected String getPageResultDestination(String viewUri, boolean error, String userId) {
+        return viewUri + "?error=" + error + "&user=" + userId;
```

Comment:
Hmm for `viewUri`, I think `parentUrl` seems better, but maybe replacing "Url" with "Uri" would be even better since [URL](https://en.wikipedia.org/wiki/URL) has quite a specific definition.